src/rust/bitbox02-rust-c/src/bitboxbase.rs

```diff
@@ -22,7 +22,6 @@
 #[allow(non_camel_case_types)]
 type c_char = u8;
 
-use bitbox02;
```

Comment:
This is problematic imho.

Just leaving the line is not good, because it doesn't do anything. Then we'd also have to strip the `bitbox02::` everywhere in the file. So the linter is okay, but you prefer to have `use bitbox02` over `bitbox02::...`.

That is possible, but then you have a bunch of constants and functions that you don't know which module they are coming from. To fix that, explicit imports would be needed, but that is also silly:

```Rust
use bitbox02::{bitboxbase_screensaver_reset, BitBoxBaseConfirmPairingRequest, CommanderError, leds_turn_small_led, leds_turn_big_led, ...........................................................}
```

if you don't mind, i would keep this one flexible, as I to do `use`  vs qualified access on a case by case basis.